docs/run/dynamic-circuits-considerations.ipynb

```diff
@@ -0,0 +1,904 @@
+    "\n",
+    "- **Limited working memory**. This primarily affects the central controllers, and jobs will fail if they cause this memory limit to be exceeded.\n",
+    "- **Latency caused by classical computation**. Running circuits that use classical feedforward and control flow involves performing classical computation during the course of the circuit execution. Due to the limited coherence time of qubits, there is a limited time budget for performing these computations. A job may fail at compile time if the compilation detects that the classical computation overhead is too large.\n",
+    "\n",
+    "The memory requirements and classical latencies of a job are affected by the following factors:\n",
+    "\n",
+    "- **Number of circuits**. When multiple circuits are submitted in a single job, they become concatenated into a single large circuit, with qubit initialization operations between them. Qubit initialization is implemented as a conditional reset on all qubits used in the large circuit.\n",
+    "  - Central controller: Memory usage scales proportionally with the number of circuits.\n",
+    "- **Amount of control flow**.\n",
+    "  - Central controller: Memory usage scales proportionally with the number of control flow decisions.\n",
+    "  - Qubit controller: A control flow construct with too many or too large logic branches may not be realizable.\n",
+    "- **Resets**.\n",
+    "  - Central controller: Memory usage scales proportionally with the number of resets.\n",
+    "- **Measurements**.\n",
+    "  - Central controller: Memory usage scales proportionally with the number of measurements used by the central controller for control flow."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the rest of this section we provide some example code that you can use to probe these limits yourself."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_ibm_provider import IBMProvider\n",
+    "\n",
+    "hub = \"ibm-q-internal\"\n",
+    "group = \"deployed\"\n",
+    "project = \"default\"\n",
+    "\n",
+    "provider = IBMProvider(instance=f\"{hub}/{group}/{project}\")\n",
+    "\n",
+    "backend_name = \"ibmq_jakarta\"\n",
+    "backend = provider.get_backend(backend_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Test `num_qubits` x `num_circuits`\n",
```

Comment:
Is there meant to be some images/outputs for these code snippets? I see a bunch of print statements and plots but no outputs. I think it would be include outputs to these code snippets if possible

Reply:
As we discussed separately, I went ahead and removed the code examples because most users can't run them anyway due to limitations on number of jobs and circuits per job.